go/install/install_darwin.go

```diff
@@ -565,25 +565,31 @@ func InstallKBFS(context Context, binPath string, force bool, timeout time.Durat
 	return nil
 }
 
-// kbnmManifestPath returns where the NativeMessaging host manifest lives on this platform.
-func kbnmManifestPath(u *user.User) string {
-	const rootPath = "/Library/Google/Chrome/NativeMessagingHosts"
-	const homePath = "Library/Application Support/Google/Chrome/NativeMessagingHosts"
+// kbnmManifestPath returns where the NativeMessaging host manifest lives on
+// this platform. Will return paths for both Chrome and Chromium.
+func kbnmManifestPaths(u *user.User) []string {
+	// Paths per https://developer.chrome.com/extensions/nativeMessaging#native-messaging-host-location-nix
 
 	if u.Uid == "0" {
```

Comment:
Hm good question. Logistically I can't think of how this would happen, but given that Google's docs have instructions for this scenario I assume it's possible? :P